deploy/sre-build-test/50-source-build-test.CronJob.yaml

```diff
@@ -119,8 +134,14 @@ spec:
 
                     if [[ -n "${JOBS_TO_DELETE}" ]] ; then
                       echo "$(date): Selected jobs for deletion: ${JOBS_TO_DELETE}"
-                      echo "${JOBS_TO_DELETE}" | xargs oc -n "${POD_NS}" delete job
+                      echo "${JOBS_TO_DELETE}" | xargs oc -n "${POD_NS}" delete job --ignore-not-found --wait=false
                     fi
+
+                    # clean up any old namespaces, other than the current one
+                    echo "$(date): Cleaning up old namespaces"
+                    oc delete ns -l "managed.openshift.io/sre-build-test,kubernetes.io/metadata.name!=${NS}" --ignore-not-found --wait=false
```

Comment:
In future we could use something like `oc get ns -l "managed.openshift.io/sre-build-test,kubernetes.io/metadata.name!=${NS}" --sort-by=.metadata.creationTimestamp` if we wanted to save the last X jobs? Not going to block merged based on this